stdlib/public/Compatibility50/Overrides.cpp

```diff
@@ -31,3 +31,8 @@ __attribute__((used, section("__DATA,__swift_hooks"))) = {
   .version = 0,
   .conformsToProtocol = swift50override_conformsToProtocol,
 };
+
+// Allow this library to get force-loaded by autolinking
+__attribute__((weak, visibility("hidden")))
```

Comment:
I don't think you can make this weak (Apple's build system will complain about it). I believe actual IRGen uses a function so it can be linkonce_odr.

Reply:
`weak` is only a problem for symbols with default visibility.

Reply:
Oh, right, good point. Withdrawn.